dev-resources/tla-demo/docker-compose.yml

```diff
@@ -0,0 +1,211 @@
+      LRSQL_ALLOW_ALL_ORIGINS: true
+    restart: always
+
+  # Noisy LRS 2
+  # - Noisy LRS 2: DB
+  nsy_lrs_2_db:
+    image: postgres:14
+    volumes:
+      - nsy_lrs_2_db_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: lrsql_user
+      POSTGRES_PASSWORD: lrsql_password
+      POSTGRES_DB: lrsql_db
+  # - Noisy LRS 2: App
+  nsy_lrs_2:
+    image: yetanalytics/lrsql:v0.7.20
+    command:
+      - /lrsql/bin/run_postgres.sh
+    ports:
+      - "8082:8080"
+    depends_on:
+      - nsy_lrs_2_db
+    environment:
+      LRSQL_API_KEY_DEFAULT: my_key
+      LRSQL_API_SECRET_DEFAULT: my_secret
+      LRSQL_ADMIN_USER_DEFAULT: my_username
+      LRSQL_ADMIN_PASS_DEFAULT: my_password
+      LRSQL_DB_HOST: nsy_lrs_2_db
+      LRSQL_DB_NAME: lrsql_db
+      LRSQL_DB_USER: lrsql_user
+      LRSQL_DB_PASSWORD: lrsql_password
+      LRSQL_POOL_INITIALIZATION_FAIL_TIMEOUT: 10000
+      LRSQL_ENABLE_REACTIONS: true
+      LRSQL_ALLOW_ALL_ORIGINS: true
+    restart: always
+
+  # Noisy LRS 3
+  # - Noisy LRS 3: DB
+  nsy_lrs_3_db:
+    image: postgres:14
+    volumes:
+      - nsy_lrs_3_db_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: lrsql_user
+      POSTGRES_PASSWORD: lrsql_password
+      POSTGRES_DB: lrsql_db
+  # - Noisy LRS 3: App
+  nsy_lrs_3:
+    image: yetanalytics/lrsql:v0.7.20
+    command:
+      - /lrsql/bin/run_postgres.sh
+    ports:
+      - "8083:8080"
+    depends_on:
+      - nsy_lrs_3_db
+    environment:
+      LRSQL_API_KEY_DEFAULT: my_key
+      LRSQL_API_SECRET_DEFAULT: my_secret
+      LRSQL_ADMIN_USER_DEFAULT: my_username
+      LRSQL_ADMIN_PASS_DEFAULT: my_password
+      LRSQL_DB_HOST: nsy_lrs_3_db
+      LRSQL_DB_NAME: lrsql_db
+      LRSQL_DB_USER: lrsql_user
+      LRSQL_DB_PASSWORD: lrsql_password
+      LRSQL_POOL_INITIALIZATION_FAIL_TIMEOUT: 10000
+      LRSQL_ENABLE_REACTIONS: true
+      LRSQL_ALLOW_ALL_ORIGINS: true
+    restart: always
+
+  # Transactional LRS
+  # - Transactional LRS: DB
+  txn_lrs_db:
+    image: postgres:14
+    volumes:
+      - txn_lrs_db_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: lrsql_user
+      POSTGRES_PASSWORD: lrsql_password
+      POSTGRES_DB: lrsql_db
+    ports:
+      - "5431:5432"
+  # - Transactional LRS: App
+  txn_lrs:
+    image: yetanalytics/lrsql:v0.7.20
+    command:
+      - /lrsql/bin/run_postgres.sh
+    ports:
+      - "8090:8080"
+    depends_on:
+      - txn_lrs_db
+    environment:
+      LRSQL_API_KEY_DEFAULT: my_key
+      LRSQL_API_SECRET_DEFAULT: my_secret
+      LRSQL_ADMIN_USER_DEFAULT: my_username
+      LRSQL_ADMIN_PASS_DEFAULT: my_password
+      LRSQL_DB_HOST: txn_lrs_db
+      LRSQL_DB_NAME: lrsql_db
+      LRSQL_DB_USER: lrsql_user
+      LRSQL_DB_PASSWORD: lrsql_password
+      LRSQL_POOL_INITIALIZATION_FAIL_TIMEOUT: 10000
+      LRSQL_ENABLE_REACTIONS: true
+      LRSQL_ALLOW_ALL_ORIGINS: true
+    restart: always
+
+  # LRSPipe
+  # - LRSPipe: Shared Redis
```

Comment:
Just to clarify, this is just simple xAPI statement aggregation, with no statement filtering whatsoever, right?

Reply:
yeah, no filters here.